src/distribute.ts

```diff
@@ -88,3 +89,94 @@ export const distributeElements = (
     );
   });
 };
+
+export const distributeLibraryItemsOnSquareGrid = (
```

Comment:
not sure where to put it, so it's here

Reply:
I think its better to put in `library.ts` since its related to library

Reply:
This function has a lot of logic so can you also add a spec for this?